dataland-e2etests/src/test/kotlin/org/dataland/e2etests/utils/ApiAccessor.kt

```diff
@@ -125,7 +125,7 @@ class ApiAccessor {
         reportingPeriod: String = "",
         ensureQaPassed: Boolean = true,
     ): List<UploadInfo> {
-        val waitTimeBeforeNextUpload = if (listOfFrameworkData.size > 1) 1000L else 0L
+        val waitTimeBeforeNextUpload = if (listOfFrameworkData.size > 1) 1L else 0L
```

Comment:
Das sollte hier bestimmt nicht reduziert werden.

Reply:
Is correct as we now only need to wait 1ms instead of 1000 to ensure that we're not running into a Error 500